src/guides/resolve-zil-domain.md

```diff
@@ -0,0 +1,203 @@
+---
+description: >-
+  This tutorial will teach you how to make a JSON RPC call to our Zilliqa smart
+  contract to get various records as well as owner address of a domain owner.
+---
+
+# Resolving .zil domain
+
+## Steps to resolve unstoppable domain
+
+In order to fetch any data from the Zilliqa blockchain you will need to do the following:
+
+1. Get a namehash of a domain
+2. Get resolver and owner addresses
+3. Get the contract sub-state related to your domain
+
+### Namehashing
+
+Namehashing is an algorithm that converts a domain name in a classical format \(like www.example.crypto\) to ERC-721 token id. This process is described  in more details over  [here](https://docs.unstoppabledomains.com/domain-registry-essentials/namehashing)
```

Comment:
I think it's also important to note the difference between namehashing in ETH and ZIL. (In ZIL we use `sha256` form `SHA-2`, instead of `keccak256`).